src/compas/geometry/primitives/circle.py

```diff
@@ -141,6 +141,14 @@ def __setitem__(self, key, value):
     def __iter__(self):
         return iter([self.plane, self.radius])
 
+    def __eq__(self, other):
+        try:
```

Comment:
why the try-except?
would just let this fail if the circle is not "properly" defined, no?

Reply:
Because something like Circle(plane, radius) == 1 should return false and not raise an exception.